bin/pre-commit

```diff
@@ -11,4 +11,4 @@ fi
 
 find . \( -name '*.php' \) -exec php -lf {} \;
 
-phpunit --filter WordPress "$phpcs_root/tests/AllTests.php"
+PHPCS_DIR=$phpcs_root phpunit --bootstrap="./Test/phpcs3-bootstrap.php" --filter WordPress "$phpcs_root/tests/AllTests.php"
```

Comment:
@westonruter Thanks for testing & correcting this.

The `PHPCS_DIR` environment variable should be only be needed when not using Composer, but having it, even when Composer was used should not cause an issue.

Or did you also get the notice while using a Composer based install ?
(and did you find the notice useful ? :smile: )